src/payments/services/payment.service.ts

```
@@ -395,23 +395,64 @@ export class PaymentService {
   }
 
+  /**
+   * Certificate + unlock using persisted intent (purpose, targets). Stripe PI webhooks often omit
+   * session metadata, so we must not rely on webhookEvent.metadata for this path.
+   */
+  private schedulePostSuccessCertificateAndUnlock(intent: PaymentIntent): void {
+    const purpose = intent.purpose;
+    const paymentIntentId = intent.id;
+    const userId = intent.userId;
+
+    if (purpose === PaymentPurpose.CERTIFICATE_BUNDLE && intent.targets?.length) {
```

ESLint

[error] Replace `purpose·===·PaymentPurpose.CERTIFICATE_BUNDLE·&&·intent.targets?.length` with `⏎······purpose·===·PaymentPurpose.CERTIFICATE_BUNDLE·&&⏎······intent.targets?.length⏎····`

(prettier/prettier)